.github/workflows/cd.yml

```diff
@@ -2,22 +2,29 @@ name: CD
 
 on:
   workflow_dispatch:
+    inputs:
+      network:
+        default: 'Mainnet'
```

Comment:
Capitalisation inconsistency: is it "mainnet" or "Mainnet"?
(Or does either work?)

Here is an example of the other capitalisation:
https://github.com/ZcashFoundation/zebra/pull/3431/files#diff-3f86e1f7ccea92fb37d223c628ee7285910e9e8308eb357e50de029965f53372R7

Reply:
Thanks for bringing this up; I was going to ask this and then I forgot. I'm not sure of the impact when executing the code, not even sure if there's a standard way to use it. cc: @dconnolly @conradoplg

Reply:
It's Mainnet because that's how the network is serialized into the config files

Reply:
![image](https://user-images.githubusercontent.com/552961/151909820-2918f5ed-8468-4f56-b885-ddbaa5da44f1.png)

Reply:
Made all terms `Mainnet`

Reply:
Actually sometimes we're using `mainnet` in the image name?

So we should check that we're not breaking anything there.